packages/groqd-playground/src/Playground.tsx

```diff
@@ -55,6 +58,14 @@ export default function GroqdPlayground({ tool }: GroqdPlaygroundProps) {
       isFetching: false,
     };
   });
+  const iframeRef = React.useRef<HTMLIFrameElement>(null);
+  const editorContainer = React.useRef<HTMLDivElement>(null);
+  const editorInitialWidth = React.useMemo(
+    () =>
+      +(localStorage.getItem(STORAGE_KEYS.EDITOR_WIDTH) || 0) ||
```

Comment:
Probably low risk, but this will still return `NaN` on the left hand if it can't coerce it with `+`

```
> +('1a' || 0)
NaN
```

Reply:
Interesting. But, luckily, `NaN || EDITOR_INITIAL_WIDTH === EDITOR_INITIAL_WIDTH`, so I think our default fallback should handle that case as well.